docs/lib/datastore/getting-started.md

```diff
@@ -9,44 +9,161 @@ Amplify DataStore provides a programming model for leveraging shared and distrib
+**Note:** this allows you to start persisting data locally to your device with DataStore, even without an AWS account.
 
 </amplify-callout>
 
-## Generate models
+## Goal
+To setup and configure your application with Amplify DataStore and use it to persist data locally on a device.
 
-Modeling your data and *generating models* which are used by DataStore is the first step to get started. GraphQL is used as a common language across all supported platforms for this process, and is also used as the network protocol when syncing with the cloud. GraphQL is also what powers some of the features such as [Automerge in AppSync](https://docs.aws.amazon.com/appsync/latest/devguide/conflict-detection-and-sync.html#automerge).
+## Prerequisites
+
+<inline-fragment platform="js" src="~/lib/datastore/fragments/js/getting-started/10_preReq.md"></inline-fragment>
+<inline-fragment platform="ios" src="~/lib/datastore/fragments/ios/getting-started/10_preReq.md"></inline-fragment>
+<inline-fragment platform="android" src="~/lib/datastore/fragments/android/getting-started/10_preReq.md"></inline-fragment>
+
+## Install Amplify Libraries
+
+<inline-fragment platform="ios" src="~/lib/datastore/fragments/ios/getting-started/20_installLib.md"></inline-fragment>
+<inline-fragment platform="android" src="~/lib/datastore/fragments/android/getting-started/20_installLib.md"></inline-fragment>
+
+There are two options to integrate the Amplify build process with the project.
+
+### Option 1: Platform integration
+
+<inline-fragment platform="js" src="~/lib/datastore/fragments/js/getting-started/30_platformIntegration.md"></inline-fragment>
+<inline-fragment platform="ios" src="~/lib/datastore/fragments/ios/getting-started/30_platformIntegration.md"></inline-fragment>
+<inline-fragment platform="android" src="~/lib/datastore/fragments/android/getting-started/30_platformIntegration.md"></inline-fragment>
+
+### Option 2: Use Amplify CLI
+
+Instead of using the platform integration, you can alternatively use the Amplify CLI on its own to accomplish the same thing that Amplify Tools is doing for you. This option is particularly useful for **existing projects** where Amplify is already configured.
+
+The base structure for a DataStore app is created by adding a new GraphQL API to your app.
+
+```console
+amplify add api
+```
+
+During the API configuration process select **GraphQL** as the API type and reply to the questions as follows. Make sure you respond **Yes, I want to make some additional changes** when prompted for **advanced settings** and turn on **conflict detection**. This setting is critical when syncing data to the cloud since the conflict resolution strategy is what allows local data to be reconciled with data from the cloud backend.
```

Comment:
I think is more **required** than critical. Because that turn on DataStore on the backend and generates the model for the clients.